Get Started with KKP on Single Node using Kubeone

archups opened the pull request:
- Documentation to demonstrate the KKP master deployment as kubeone addon on single node using kubeone.

Signed-off-by: archups <archupsawant@gmail.com>

archups commented:
/assign @scheeles for review!

stroebitzer commented on content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:
```diff
@@ -0,0 +1,96 @@
+
+# Get started with KKP - “KKP on single node with Kubeone”
+
+This chapther will guide you through the KKP Master setup on a single master/worker k8s node using Kubeone. Here, we are leveraging the kubeone addons capability for quick installation of KKP.
```
IMO chapther should be chapter.

archups replied:
Thanks @stroebitzer. Fixed the typo error!

scheeles commented:
/assign @kron4eg @xmudrii

archups commented on content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:
```diff
@@ -0,0 +1,96 @@
+enableToc = true
++++
+
+# Get started with KKP - “KKP on single node with Kubeone”
```
Removed the redundant title.

archups commented:
> I think this should be a rather kubeone doc where a person could init a 1 node cluster, ragardless of KKP.
> 
> And that later we should link to that doc from KKP installation.

@kron4eg @scheeles @stroebitzer @xmudrii - Open to suggestion on where to place this guide. Can we finalize it, if we are good with content?

scheeles commented:
@shaase-ctrl PTAL

shaase-ctrl commented on content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:
```diff
@@ -0,0 +1,97 @@
+* Update DNS mapping with External IP of the nginx ingress controller service. Incase of AWS, the CNAME mapping for $TODO_DNS with External IP should be created for resolution.
+
+* Nginx Ingress Controller Load Balancer configuration - Add the node to backend pool manually.
+> Should be supported in future as part of Feature request[#1822](https://github.com/kubermatic/kubeone/issues/1822)
```
... in the future

archups replied:
Done!

shaase-ctrl commented on content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:
```diff
@@ -0,0 +1,97 @@
++++
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone”"
```
Suggestion: Get started with KKP - KKP on a single node with KubeOne on AWS

We could copy this and make guides for digitalocean, azure etc. This kind of content would help a lot.

archups replied:
@shaase-ctrl  Updated the title as per Suggestion

shaase-ctrl commented:
> 

Yes and no. From my perspective this helps getting it on with KKP and KubeOne is just a tool to do that. Another entry should be single node kubeone clusters

archups commented on content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:
````diff
@@ -0,0 +1,97 @@
+  ```bash
+  kubectl get certificates -A -w
+  ```
+  > Wait for while, if still kubermatic-api-xxx pods / kubermatic & dex tls certificates are not in Ready state, delete  and wait to get validated.
````
taken care!

stroebitzer commented:
@shaase-ctrl where should we put this docu? KKP or KubeOne?

shaase-ctrl commented:
@stroebitzer KKP Docu of course. But we can recycle parts within KubeOne as well - double tap.

archups commented:
@shaase-ctrl can you please merge this PR as we have addressed comments? Can you help with moving https://github.com/kubermatic-labs/kkp-on-node repository under "kubermatic" organization. Thank you

shaase-ctrl commented:
I think you need to open another PR to the correct repository @archups

archups commented:
@shaase-ctrl  can you please assist with which repository we need to place this installation document?

shaase-ctrl commented:
/lgtm

shaase-ctrl commented:
/approve

kron4eg commented:
/approve

kron4eg commented:
/lgtm
